draft-ietf-tls-tls13.md

```diff
@@ -3372,8 +3372,7 @@ that are not known to be supported by the client. This fallback chain MAY
 use the deprecated SHA-1 hash algorithm only if the "signature_algorithms"
 extension provided by the client permits it.
 If the client cannot construct an acceptable chain using the provided
-certificates and decides to abort the handshake, then it MUST abort the
```

Comment:
The "decides to abort the handshake" bit was the result of long mailing list discussions on various different use cases and should not be removed.